tinyiiod.h

```diff
@@ -24,7 +24,7 @@ struct tinyiiod;
 
 struct tinyiiod_ops {
 	/* Read from the input stream */
-	void (*read)(char *buf, size_t len);
+	int32_t (*read)(char *buf, size_t len);
```

Comment:
this change `libtinyiiod: Modify return value for consistency ` is nice;
it shouldn't break API too bad; it would cause compiler warnings though, but we can live with that;

i would change the name to `libtinyiiod: add return value to read() callback` so that the title better reflects what it's doing